jp/components/list.md

```diff
@@ -81,10 +81,10 @@ export class AppModule {}
 <!--contacts.component.html-->
 
 <igx-list>
-    <igx-list-item isHeader="true">ヘッダー</igx-list-item>
-    <igx-list-item>項目 1</igx-list-item>
-    <igx-list-item>項目 2</igx-list-item>
-    <igx-list-item>項目 3</igx-list-item>
+    <igx-list-item isHeader="true">Header</igx-list-item>
```

Comment:
In case there is any question, I did this on purpose so that it would match the sample displayed below it.  The samples are not internationalized so they will all appear in English.